tools/osbuilder/rootfs-builder/init_trusted_storage.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+[ -n "${BASH_VERSION:-}" ] && set -o errtrace
```

Comment:
What is the `BASH_VERSION` test for?

Rationale: I'm trying to minimize the possible path variations in the behavior of a critical script. IMO, we want to either have tests checking the correct behavior in both alternatives, or else, not have the test.

Reply:
I think this can be simplified to `set -o errtrace` here. I tend to check `BASH_VERSION` to allow a bash script to be sourced by zsh for example, but that doesn't apply here.